web/src/compose_recipient.js

```diff
@@ -135,6 +172,8 @@ function get_options_for_recipient_widget() {
             }
             return 0;
         });
+    options.unshift({name: PRIVATE_MESSAGE, value: PRIVATE_MESSAGE});
```

Comment:
The `name` here isn't tagged for translation; it'd be better to just spell it out explicitly here. Also how does this related to the TODO at the top of the `new_value === PRIVATE_MESSAGE` block?

Reply:
updated the TODO to be more clear:

```
        // TODO: When "Direct Message" is selected, we show "DM" on the dropdown
        // button. It would be nice if the dropdown supported a way to attach
        // the "DM" button display string so we wouldn't have to manually change
        // it here.
```

Reply:
Looks great.

Reply:
OK but `name: PRIVATE_MESSAGE` still needs to be `name: // a translated string`

Reply:
is it not? https://github.com/zulip/zulip/pull/25133/files#diff-b0b7a410e188c176844d90cfde67ba408f596e40d11e94ab24d6b8278cc82f4dR182

Reply:
Looks like it is now.